tree-sitter-usfm3/grammar.js

```diff
@@ -369,10 +370,10 @@ module.exports = grammar({
       $.tc,
       $.tcr))
     )),
-    thTag: $=> /\\th([12345](-[12345])?)?/,
-    thrTag: $=> /\\thr([12345](-[12345])?)?/,
-    tcTag: $=> /\\tc([12345](-[12345])?)?/,
-    tcrTag: $=> /\\tcr([12345](-[12345])?)?/,
+    thTag: $=> /\\th(\d+(-\d+)?)?/,
```

Comment:
This is a new realization for me that this was possible. See e.g. https://docs.usfm.bible/usfm/3.1/char/tables/thr.html